source/server/getting-started/migrating.rst

```diff
@@ -127,7 +127,7 @@ When finding replacements for your Bukkit plugins, there are a few points to kee
 * Some Sponge plugins that are ported from Bukkit may change in functionality, or may not even use the same
   configuration structure.
 
-Migrating from Canary
+Migrating from CanaryMod
 ~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Please extend the `~` line to the same length as the title right above it.

Reply:
Okay. I did that